src/borg/item.pyx

```diff
@@ -304,6 +307,15 @@ class EncryptedKey(PropDict):
     salt = PropDict._make_property('salt', bytes)
     hash = PropDict._make_property('hash', bytes)
     data = PropDict._make_property('data', bytes)
+    argon2_time_cost = PropDict._make_property('argon2_time_cost', int)
+    argon2_memory_cost = PropDict._make_property('argon2_memory_cost', int)
+    argon2_parallelism = PropDict._make_property('argon2_parallelism', int)
+    argon2_type = PropDict._make_property(
+        'argon2_type',
+        argon2.low_level.Type,
+        encode=lambda t: t.name.encode(),
+        decode=lambda b: argon2.low_level.Type[b.decode()]
+    )
```

Comment:
quite some effort.

but considering key files will last for long, we can not accept any library-side change of `argon2.low_level.Type` anyway (because then our existing keyfiles would not match any more and stuff would likely blow up).

so just have str / bytes or just always use `ID`?